sql-plugin/src/main/330+/scala/org/apache/spark/RapidsSparkIndexOutOfBoundsException.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark
+
+import ai.rapids.cudf.ColumnVector
+
+object RapidsSparkIndexOutOfBoundsException {
+  def throwSparkArrayIndexOutOfBoundsException(
+      ordinalValue: Int,
+      minNumElements: Int): ColumnVector = {
+    throw new org.apache.spark.SparkArrayIndexOutOfBoundsException(
```

Comment:
Shouldn't this be using the public `QueryExecutionErrors.invalidArrayIndexError` method instead of trying to access a private class directly?